src/DocTests.jl

```diff
@@ -239,7 +239,9 @@ function eval_script(block, sandbox, meta::Dict, doc::Documents.Document, page)
     #
     #
     #       to mark `input`/`output` separation.
-    input, output = split(block.code, "# output\n", limit = 2)
+
+    # Append "\n" before split in case of empty output
+    input, output = split(block.code*"\n", "# output\n", limit = 2)
```

Comment:
Perhaps it is a bit cleaner to split by a regex
```suggestion
    input, output = split(block.code, r"^# output$"m, limit = 2)
```

Reply:
Very nice!